app/services/notifications.rb

```diff
@@ -29,4 +29,12 @@ def self.comment_data(comment)
       }
     }
   end
+
+  def self.moderator_availability_delay
+    28.hours
```

Comment:
I understand the need to label "28 hours" with a meaningful name, but wouldn't it be simpler to use a a constant in `Notifications::Moderation` ?